modules/microshift-ingress-controller-config.adoc

```diff
@@ -0,0 +1,153 @@
+
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-ingress-controller.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-ingress-control-config_{context}"]
+= Configuring ingress control in {microshift-short}
+
+You can use detailed ingress control settings by updating the {microshift-short} service configuration file.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have root access to the cluster.
+* Your cluster uses the OVN-Kubernetes network plugin.
+//Q: Any other prereqs?
+
+.Procedure
+
+. You can apply ingress control settings in either of the two following places:
+
+.. Update the {microshift-short} `config.yaml` configuration file by making a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory.
+
+.. Use a configuration snippet to apply the ingress control settings you want. To do this, create a configuration snippet YAML file and put it in the the `/etc/microshift/config.d/` configuration directory.
+
+[NOTE]
+====
+* After you create it, the `config.yaml` file takes precedence over built-in settings. The configuration file is read every time the {microshift-short} service starts.
+* Configuration snippet YAMLs take precedence over all settings anda `config.yaml` configuration file. See the Additional resources links for more information.
+====
+
+. When using the `config.yaml`, replace the default values in the `network` section of the {microshift-short} YAML with your valid values.
++
+.Ingress controller configuration field snippet with default values
+[source,yaml]
+----
+apiServer:
+# ...
+ingress:
+    defaultHTTPVersion: 1
+    forwardedHeaderPolicy: Append
+    httpCompression:
+        mimeTypes:
+            - ""
+    httpEmptyRequestsPolicy: Respond
+# ...
+    logEmptyRequests: Log
+# ...
+    tuningOptions:
+        clientFinTimeout: 1s
+        clientTimeout: 30s
+        headerBufferBytes: 0
+        headerBufferMaxRewriteBytes: 0
+        healthCheckInterval: 5s
+        maxConnections: 0
+        serverFinTimeout: 1s
+        serverTimeout: 30s
+        threadCount: 4
+        tlsInspectDelay: 5s
+        tunnelTimeout: 1h
+# ...
+----
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`defaultHTTPVersion`
+|`defaultHTTPVersion` sets the HTTP version for the Ingress Controller. Default value is `1`.
```

Comment:
Just a suggestion to consider - for each description like this one that starts with the name of the parameter that's already mentioned in the adjacent cell, I feel like you could remove the name from the start of the description. This way, you'd avoid the slightly awkward formatting of starting a sentence with a lowercase letter because it's a parameter name:
```suggestion
|Sets the HTTP version for the Ingress Controller. Default value is `1`.
```

Up to you though